lib/flutter_chips_selector.dart

```diff
@@ -69,6 +69,16 @@ class ChipsSelectorState<T> extends State<ChipsSelector<T?>> {
 
   late VoidCallback _currentFocusNodeListener;
 
+  final FocusNode _focusableActionDetectorFocusNode = FocusNode(
+    debugLabel: "#chip selector: focusableActionDetector focus node",
+    skipTraversal: true,
+  );
+
+  final FocusNode _rawKeyboardListenerFocusNode = FocusNode(
+    debugLabel: "#chip selector: rawKeyboardListener focus node",
+    skipTraversal: true,
+  );
```

Comment:
You need to dispose these

Reply:
ah, true. Thanks for noticing that. Done.